Remove double import of Font Awesome

MyGiHu opened the pull request.

victoriadrake commented:
Is the double import in this file? I don't see it. Would you please add a comment to expain?

MyGiHu commented:
The double import of Font Awesome happens through two files. ["style.sass"](https://github.com/victoriadrake/hugo-theme-introduction/blob/7bebd01f426deed30f6eaeef582bc417ea70191c/assets/sass/style.sass) and ["_fonts.sass"](https://github.com/victoriadrake/hugo-theme-introduction/blob/7bebd01f426deed30f6eaeef582bc417ea70191c/assets/sass/_fonts.sass).

"style.sass" imports "_fonts.sass" which contains a command to import Font Awesome. A few lines later "style.sass" imports Font Awesome again. One import should be sufficient and reduzes the file size of the to creating theme css file.

Note: Font Awesome is imported multiple times by "style.sass" for creating a dark and light (auto option) or user selected style theme. This is not topic of this pull request.

Simplified example:
style.sass:
`@import "fonts"`
`@import "fontawesome-import"`

_fonts.sass:
`@import "fontawesome-import"`

I removed the redundant import command in "_fonts.sass" because "style.sass" contains a neat arranged import list.

victoriadrake commented:
Thank you again!
